### PR TITLE
Fix CircleCI build

### PR DESCRIPTION
**Pull request by Geod24**

### CODE REVIEW

**Comment by Geod24** [comment]
This is green, can I get a review ?

File: .travis.yml [hunk 1]
```diff
@@ -20,10 +20,6 @@ matrix:
     - d: ldc-1.16.0
```
Comment (TrustHenry) [inline .travis.yml:20]: ldc-1.17.0 Is it okay not to do a version-up here?

**Review by TrustHenry — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (Geod24) [reply .travis.yml:20]: The LDC upgrade is here: https://github.com/bpfkorea/agora/pull/283
And it needs a bit more work.

Comment (TrustHenry) [reply .travis.yml:20]: Yes, I checked.

**Comment by Geod24** [comment]
This was part of https://github.com/bpfkorea/agora/issues/269

**Pull request merged**